cli/src/args.ts

```diff
@@ -77,10 +85,34 @@ export const args: BenchmarkArguments = {
       task.trainingInformation.batchSize = unsafeArgs.batchSize;
       task.trainingInformation.roundDuration = unsafeArgs.roundDuration;
       task.trainingInformation.epochs = unsafeArgs.epochs;
+      task.trainingInformation.validationSplit = unsafeArgs.validationSplit;
 
       // For DP
-      // TASK.trainingInformation.clippingRadius = 10000000
-      // TASK.trainingInformation.noiseScale = 0
+      const {dpDefaultClippingRadius, noiseScale, delta} = unsafeArgs;
+
+      if (
+        dpDefaultClippingRadius !== undefined ||
+        noiseScale !== undefined ||
+        delta !== undefined
+      ){
+        if(
+          task.trainingInformation.scheme === "decentralized" ||
+          task.trainingInformation.scheme === "federated"
+        ){
+          task.trainingInformation.privacy ??={};
+          const privacy = task.trainingInformation.privacy;
+
+          if (dpDefaultClippingRadius !== undefined){
+            privacy.dpDefaultClippingRadius = dpDefaultClippingRadius;
+          }
+          if (noiseScale !== undefined){
+            privacy.noiseScale = noiseScale;
+          }
+          if (delta !== undefined){
+            privacy.delta = delta;
+          }
```

Comment:
setting a field to undefined is akin to not setting it at all (at least in our handling of it)

```suggestion
          task.trainingInformation.privacy = {
            dpDefaultClippingRadius,
            noiseScale,
            delta,
          };
```